test/evergreen.yml

```diff
@@ -4568,6 +4568,14 @@ tasks:
                 cat $t && rm $t
             fi
 
+            # Code complexity statistics
+
+            python3 "../metrixplusplus/metrix++.py" view --format=python > code_complexity_summary.txt
+            python3 "../metrixplusplus/metrix++.py" export --db-file=metrixpp.db > metrixpp.csv
```

Comment:
python3 --> python to be consistent with existing metrix++.py calls above.

Reply:
code_complexity_summary.txt --> code_complexity_summary.json to reflect the format of the output.

Reply:
Fixed in 7cc186a02197e388c9a2fcaa97331d24c84376a5